normandy/classifier/admin/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Import views so that admin autodiscover picks them up.
+import normandy.classifier.admin.views
```

Comment:
Sort've. The problem is whether `AppConfig.ready` runs before the main urlconf is constructed, because that's when the admin URLs are generated and all ModelAdmins need to be registered before that point. `autodiscover` simply imports all the `app.admin` modules it can find, and which includes this `__init__` file.

Even if we had an `AppConfig` running `autodiscover` we'd have to do this import anyway. It's the one downside to having `admin` be a package instead of a module, and I think it's fine.